tests/symbolics/test_sympy_backend.py

```diff
@@ -35,6 +35,33 @@ def test_is_sympy_int(expression, expected):
     assert sympy_backend.is_constant_int(expression) == expected
 
 
+@pytest.mark.parametrize(
+    "expression_str, expected",
+    [
+        # Known constants
+        ("pi", pi),
+        ("PI", pi),
+        ("e", E),
+        ("E", E),
+        ("infinity", oo),
+        ("Infinity", oo),
+        # Expressions with constants
+        ("2 * pi", 2 * pi),
+        ("e + 1", E + 1),
```

Comment:
I'd add some cases where the constants are parts of a more complicated expression – e.g. input to a function `sin(0.25*pi)`